.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -72,6 +72,27 @@ on:
         description: The artifact name of the signed provenance. The file must have the .intoto extension. Defaults to <filename>.intoto for single artifact or multiple.intoto.jsonl for multiple artifacts.
         required: false
         type: string
+      upload-assets:
+        description: >
+          If true, provenance is uploaded to a GitHub release for new tags.
+          When the workflow does not run on a new tag, such as on a workflow_dispatch,
+          the `upload-tag-name` argument must be provided as well.
```

Comment:
This kind of makes it sound like it will fail if `upload-assets` is true and `upload-tag-name` is not provided. I think in reality we just silently ignore it?

Reply:
True, we just skip asset upload... Hmm I'll reword this.